README.md

```diff
@@ -10,12 +10,23 @@ Supported versions: 1.2 and 1.3.
 
 ## Building and running
 
-### Building
+### Prerequisites:
 
-```bash
-make
-```
+* [Go compiler](https://golang.org/dl/)
 
+### Building
+
+1. Get the code from the Percona repository:
+ 
+    ```bash
+    go get github.com/percona/proxysql_exporter
+    ```
+ 2. Switch to the buld directory and run ``make``:
+ 
+    ```bash
+    cd ${GOPATH-$HOME/go}/src/github.com/percona/proxysql_exporter
+    make
```

Comment:
That will not only build exporter but also run tests, etc. I'm not sure if that what you want.

Reply:
Possibly we could use `make format build` instead, but single `make` looks very familiar, and just spending a moment to run tests does nothing bad...